common/src/commonMain/kotlin/entity/DiscordEmoji.kt

```diff
@@ -51,3 +51,11 @@ public data class DiscordPartialEmoji(
     val name: String? = null,
     val animated: OptionalBoolean = OptionalBoolean.Missing,
 )
+
+/**
+ * List of application [emojis][DiscordEmoji].
+ *
+ * @property items the list of emojis
+ */
+@Serializable
+public data class ApplicationEmojis(val items: List<DiscordEmoji>) : List<DiscordEmoji> by items
```

Comment:
`equals` and `hashCode` produced by data classes don't conform to the requirements of `List`, I suggest to simply not implement `List`.

Reply:
Or better remove this class as a whole.

Reply:
Nope, don't remove, discord returns a JSON object with `items` in it.

Reply:
We could just make it a value class instead of a data class

Reply:
that doesn't fix it either